src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -135,7 +135,7 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
     }
 
     public List<String> listProcess(@Nonnull EnvVars launchEnv, @Nonnull String containerId) throws IOException, InterruptedException {
-        LaunchResult result = launch(launchEnv, false, "top", containerId);
+        LaunchResult result = launch(launchEnv, false, "top", containerId, "-o","pid,uid,time,command","-ax");
```

Comment:
we actually only look for `command`, so could be simplified.
Such a pity `docker top` isn't normalizing returned data :-\